translations/ui-plugin-find-import-profile/en_GB.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
As far as I remember there is no need really for adding all translations files for a new module. en.json would be enough, other files will be automatically generated. @dltj could you please clarify this.

Reply:
Yes -- that's correct.  Just `en.json` is needed.  The rest will be automatically generated.  Details at https://dev.folio.org/guides/commence-a-module/#front-end-translations

Reply:
Done